site/pages/404.js

```diff
@@ -0,0 +1,9 @@
+import React, { PureComponent } from 'react';
+
+export default class NotFound extends PureComponent {
+  render() {
+    return (
+      <div>Page not found.</div>
+    );
+  }
+}
```

Comment:
This page might not be necessary. If you hit a 404 on mapbox.com/assembly won't you land on the 404 page for mapbox.com?